frontend/.eslintrc.js

```diff
@@ -14,51 +14,22 @@ module.exports = {
         ecmaVersion: 11,
     },
     extends: [
-        "eslint:all",
+        "eslint:recommended",
     ],
     rules: {
         // Turn off or reconfigure some rather opinionated and intrusive rules
```

Comment:
This is now outdated.

```suggestion
```

Reply:
That has already been removed. And yeah I know I'm at fault for recommending reviewing commit by commit :P

Reply:
Ah ok, cool! :)